src/kaleidoscope/plugin/Model01-TestMode.cpp

```diff
@@ -14,6 +14,8 @@
  * this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+#ifndef ARDUINO_VIRTUAL
```

Comment:
Is the HardwareTestMode a replacement for Model01-TestMode? It so, it would make sense to replace its use in the stock firmware.

Reply:
It is. It should.

Reply:
Are you ok with me preparing a PR that does the replacement in Model01-Firmware?